compiler/executable/match_/planner/plan.rs

```diff
@@ -444,6 +443,7 @@ impl<'a> ConjunctionPlanBuilder<'a> {
                 self.graph.variable_index[variable]
             })
             .collect();
+        // TODO: This is just a mock
```

Comment:
... and what (when?) do we want to do with it in the end? 

It's a common issue with codebases to put TODOs "for the next week" and abandon them for ages. After some time, it's really hard to restore the context. I'd try to be more descriptive for other developer's sake

Reply:
There are better examples of your todos in other files. Let's put a direct command to action / a desired result: "Make it ***", "implement ***", "should use ***"